test/integration/serverless/src/go.mod

```diff
@@ -1,64 +1,77 @@
 module lambda-time
 
-go 1.18
+go 1.21
+
+toolchain go1.22.8
 
 require (
-	github.com/DataDog/datadog-lambda-go v1.11.1-0.20231030204701-7ec92619787d
-	github.com/aws/aws-lambda-go v1.34.1
-	gopkg.in/DataDog/dd-trace-go.v1 v1.54.1
+	github.com/DataDog/datadog-lambda-go v1.20.0
```

Comment:
I'm pretty sure the failure in the serverless integration tests is due to a bump in the version of datadog-lambda-go. I understand that the current version is quite out of date, but I would recommend reverting this whole file to its original state to make testing easier.